static/app/views/performance/transactionSummary/transactionTags/tagValueTable.tsx

```diff
@@ -172,7 +172,17 @@ export class TagValueTable extends Component<Props, State> {
           handleCellAction={this.handleCellAction(column, dataRow.tags_value, actionRow)}
           allowActions={allowActions}
         >
-          <TagValue row={dataRow} />
+          {column.name === 'release' ? (
+            <Link
```

Comment:
@0Calories You will need to pass in the project query string in as well since this should persist once you're on the transaction summary page. Otherwise multiple projects could share the same release name.

See: https://github.com/getsentry/sentry/blob/24abd344410004a617f276105d9ddb7bfaa54712/static/app/views/eventsV2/table/tableView.tsx#L405

Reply:
@dashed Is it ok to simply pick the project id off the query string in the current Location object, or would I have to do it the same way as in the example you linked?

I was looking at an example here which seems a bit simpler: 

https://github.com/getsentry/sentry/blob/06051ffdfa106ab1eaa8ba6b9015630fee720c41/static/app/views/dashboardsV2/widget/metricWidget/index.tsx#L76